player/lua/osc.lua

```diff
@@ -1866,7 +1869,13 @@ local function osc_init()
         return ("\238\132\134" .. osc_styles.smallButtonsLlabel .. " " ..
                (mp.get_property_native("aid") or "-") .. "/" .. audio_track_count)
     end
-    ne.eventresponder["mbtn_left_up"] = command_callback(user_opts.audio_track_mbtn_left_command)
+    if audio_track_count == 1 and
+       user_opts.audio_track_mbtn_left_command == default_audio_command then
```

Comment:
this is the worst possible approach
I would either:
1. redefine `audio_track_mbtn_left_command` to be "the command that runs to select between 2 or more tracks"
2. or have the setting be empty by default and use that to switch between the special and user-configured cmd